concrete/src/Package/Dependency/MissingRequiredPackageException.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Interesting. I ran php-cs-fixer yesterday and it removed this blank line instead of adding it (https://github.com/concrete5/concrete5/pull/7370/commits/997a102f20374f6ed404f5a141369930f8a72a29#diff-1d7786ac0a9226a4276e4b12b903da4bL2)

Reply:
Well, I guess you are using the [`.php_cs.dist` file](https://github.com/concrete5/concrete5/blob/0c0f0c41174324410d24303010752334b0598078/.php_cs.dist) included in concrete5, don't you?

I'm using a hacky version of #7355.

Reply:
> Well, I guess you are using the .php_cs.dist file included in concrete5, don't you?

Yes, but I (also?) prefer the blank line after the namespace ;)

Reply:
Me too: I think that's a really important info that deserves to stay alone, with blank lines before and after it.

Reply:
We should follow [PSR-12](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#3-declare-statements-namespace-and-import-statements) on this:

> The header of a PHP file may consist of a number of different blocks. If present, each of the blocks below MUST be separated by a single blank line, and MUST NOT contain a blank line.